doc/api/test.md

```diff
@@ -2086,8 +2086,10 @@ added: v22.3.0
 
 This function is used to mock the exports of ECMAScript modules, CommonJS
 modules, and Node.js builtin modules. Any references to the original module
-prior to mocking are not impacted. The following example demonstrates how a mock
-is created for a module.
+prior to mocking are not impacted. In order to enable module mocking, Node.js must
+be started with the [`--experimental-test-module-mocks`][] command-line flag.
+
+The following example demonstrates how a mock is created for a module.
```

Comment:
IIRC any conditions for this belong in the stability block suite

Reply:
I tried to make this consistent (within the test docs page, at least); other examples using experimental flags, neither of which does so in the blockquote:

- https://nodejs.org/api/test.html#snapshot-testing (I based the sentence I added on this); and
- https://nodejs.org/api/test.html#collecting-code-coverage